WORKSPACE

```diff
@@ -3,4 +3,15 @@ workspace(name = "io_tweag_rules_haskell")
 local_repository(
   name = "examples",
   path = "examples"
-)
\ No newline at end of file
+)
+
+http_archive(
+    name = "io_tweag_rules_nixpkgs",
+    strip_prefix = "rules_nixpkgs-a300f574885c50430147e457d21ec22a9fe015f4",
+    urls = ["https://github.com/tweag/rules_nixpkgs/archive/a300f574885c50430147e457d21ec22a9fe015f4.tar.gz"],
```

Comment:
In other repos, I like to add a comment saying why I'm using a particular SHA1. e.g.

```python
# latest master at the time of last update
```

Or

```python
# Must be later than 0.x in order to get $bugfix
```

Worth doing that here?